src/OpenRoad.i

```diff
@@ -256,6 +258,24 @@ using odb::dbTech;
   $1 = sta::tclListSeqLibertyCell($input, interp);
 }
 
+%typemap(in) vector<const char*> * {
```

Comment:
Can you just use
```
%include <std_vector.i>
%template std::vector<const char*>
```

Reply:
I tried, but I kept getting an error that some functions where getting redefined during linking. I would prefer that solution since it's cleaner and easier to maintain, so I would welcome suggestions to how to approach this.

Reply:
I filed https://github.com/swig/swig/issues/2305 as I think this is a swig bug.  I'll approve for now and hope to revisit later.